R/billingInfo.R

```diff
@@ -0,0 +1,137 @@
+#' @title getBillingInfo
+#' @description Get billing account information
+#' @param asDataFrame Output as data.frame, Default: TRUE
+#' @param userId User ID
+#' @return Billing account information in list or data.frame output
+#' @rdname getBillingInfo
+#' @export 
+#' @importFrom tibble as_tibble
+
+getBillingInfo <- function(asDataFrame = TRUE, userId) {
```

Comment:
Would it be possible to keep the function names consistent with the API documentation and the other methods. So this would be `getBillingAccount` and the others would be `getBillingAccountDatabases` etc.. ?